bcauth/jinja2/oauth2_provider/application_confirm_delete.html

```diff
@@ -0,0 +1,18 @@
+{% extends "account/profile.html" %}
+
+{% block head_subtitle %} - {{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+{% block body_title %}{{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+
+{% block content %}
+    <h3>{{ _("Are you sure to delete the application %(app_name)s?") | format(app_name=application.name) }}</h3>
```

Comment:
@jwhitlock and I talked on IRC about this. The jinja environment that bc is using is missing some bits as noted in pull #61. We're going to create a bug to switch to address extracting strings from Jinja templates which will probably involve changes to how the Jinja environment is being built and escaping and all that. That'll get done at some point in the future and not be part of this PR.